### ISSUE TITLE
WASM build broken on newest nightly for MacOS

### ISSUE THREAD

**Issue by jnferner** [rust-lang/rust#54027]
running `cargo build --target wasm32-unknown-unknown` on `rustc 1.30.0-nightly (a8c11d216 2018-09-06)` returns the following:

```
error: linking with `rust-lld` failed: signal: 6
note: dyld: Library not loaded: @rpath/libLLVM.dylib
            Referenced from: /Users/jnf/.rustup/toolchains/nightly-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/bin/rust-lld
            Reason: image not found
```

Might be related to https://github.com/rust-lang/rust/issues/53813

**Comment by matthiaskrgr** [comment]
I think this might be fixed in the next nightly: https://github.com/rust-lang/rust/pull/54018

**Comment by jnferner** [comment]
Yeah, that looks pretty relevant. Thanks a bunch!

**Comment by rbalicki2** [comment]
I can confirm this is fixed on `rustc 1.30.0-nightly (fc81e3624 2018-09-07)`
